File: Tutorial3SuperClassEntities/src/main/java/com/example/tutorial3superclassentities/data/source/local/MeasurementAnglemeterDao.java
package com.example.tutorial3superclassentities.data.source.local;

import android.arch.persistence.room.Dao;
import android.arch.persistence.room.Query;

import com.example.tutorial3superclassentities.data.source.model.MeasurementAnglemeter;

import java.util.List;

@Dao
public interface MeasurementAnglemeterDao extends BaseDao<MeasurementAnglemeter> {
    @Query("SELECT * FROM table_clinometer_angle")
   List<MeasurementAnglemeter> getAll();

    @Query("DELETE  FROM table_clinometer_angle")
   int deleteAll();

}

File: Tutorial3-1SuperClassEntities/src/main/java/com/example/tutorial3superclassentities/data/source/local/MeasurementAnglemeterDao.kt
package com.example.tutorial3superclassentities.data.source.local

import androidx.room.Dao
import androidx.room.Query
import com.example.tutorial3superclassentities.data.source.model.MeasurementAnglemeter

@Dao
interface MeasurementAnglemeterDao : BaseDao<MeasurementAnglemeter> {

    @Query("SELECT * FROM table_clinometer_angle")
    fun getAll(): List<MeasurementAnglemeter>

    @Query("DELETE  FROM table_clinometer_angle")
    fun deleteAll(): Int
}